### PR TITLE
Add a basic test for a rust postgres client

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: tests/client_tests/rust/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rust-cratedb-tests"
+version = "0.1.0"
+authors = ["Mathias Fussenegger <mathias@crate.io>"]
+edition = "2018"
```
Comment (seut) [inline tests/client_tests/rust/Cargo.toml]: not this year? ;)

File: tests/client_tests/rust/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    let x = 20;
+    let name = "Trillian";
+    client.execute(&stmt, &[&id, &x, &name]).unwrap();
+    client.execute("refresh table tbl", &[]).unwrap();
+
+    for row in client.query("SELECT id, x, name FROM tbl", &[]).unwrap() {
+        let id: i32 = row.get("id");
+        let x: i32 = row.get("x");
+        let name: &str = row.get("name");
+        println!("id={} x={} name={}", &id, &x, &name);
```
Comment (seut) [inline tests/client_tests/rust/src/main.rs]: shall we maybe assert for valid rows, e.g. something like `assert!(&id > 0)`?

File: tests/client_tests/rust/Cargo.toml (hunk 1 above)
Comment (mfussenegger) [reply tests/client_tests/rust/Cargo.toml]: This was generated by `cargo new`, not sure why it took last year :) 

Will change it.

File: tests/client_tests/rust/src/main.rs (hunk 2 above)
Comment (mfussenegger) [reply tests/client_tests/rust/src/main.rs]: yep will do

File: tests/client_tests/rust/Cargo.toml (hunk 1 above)
Comment (mfussenegger) [reply tests/client_tests/rust/Cargo.toml]: Edition is actually cargo specific.
```
Caused by:
  failed to parse the `edition` key

Caused by:
  supported edition values are `2015` or `2018`, but `2019` is unknown
```

**Comment by mfussenegger** [comment]
retest this please

**Pull request merged**